lib/provider.coffee

```diff
@@ -135,11 +161,18 @@ class BibtexProvider
 
   prettifyTitle: (title) ->
     return if not title
-
-    if (colon = title.indexOf(':')) isnt -1 and title.words().length > 5
+    if (colon = title.indexOf(':')) isnt -1 and title.split(" ").length > 5
       title = title.substring(0, colon)
 
-    title.titleize().truncateOnWord 30, 'middle'
+    # make title into titlecaps, trim length to 30 chars(ish) and add elipsis
```

Comment:
This does a poor job of replicating the Sugar.js behavior. I think it's important to show words from both the beginning and the end of the title, since, in some fields, titles commonly begin with the same or similar words.